server/sonar-web/src/main/js/apps/projectActivity/components/EventInner.js

```diff
@@ -35,7 +35,7 @@ export default function EventInner(props /*: { event: EventType } */) {
             className={'project-activity-event-icon margin-align ' + event.category}
           />
         </div>
-        <span className="project-activity-event-inner-text">
+        <span className="project-activity-event-inner-text text-ellipsis" title={event.name}>
```

Comment:
Maybe a tooltip should be added then. Because right now there is now way to have the full event description. I think I preferred like it was before.

Reply:
Yeah, I didn't think that it would impact all event, and not only custom ones. I'll do something different then.